cli/inagoctl.go

```diff
@@ -34,19 +38,34 @@ var (
 			// This callback is executed after flags are parsed and before any
 			// command runs.
 
+			logLevel := "INFO"
+			if globalFlags.Verbose {
+				logLevel = "DEBUG"
+			}
+
+			logger := requestcontext.MustGetLogger(
+				requestcontext.LoggerConfig{
+					Name:                "inago",
+					Level:               logLevel,
+					Color:               isatty.IsTerminal(os.Stderr.Fd()),
+					IncludeNameInFormat: false,
+				},
+			)
+			newLogger = &logger
```

Comment:
Why do you even want to do this pointer redirection? Make `newLogger     *requestcontext.Logger` to `newLogger     requestcontext.Logger` and you are done.